src/txn/txn.c

```diff
@@ -1640,3 +1640,5 @@ __wt_verbose_dump_txn(WT_SESSION_IMPL *session)
 
 	return (0);
```

Comment:
@nehakhatri5 Do you think we can enhance this portion?

Reply:
How could this be set?  We start with zeroed memory and only set the fields we want.                                                                                 
I understand wanting the comment, maybe it can go near line 1716?